src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        int randomNumber = (random.nextInt(100));
+        String randomColor = getRandomColor();
+        Ball ball = new Ball(randomColor, randomNumber);
+        return ball;
+    }
```

Comment:
```suggestion
public class Lottery {
    private Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();
    
    ...
```

нам не нужно екстендить класс ColorSupplier тут, классы Lottery и ColorSupplier не имеют общего состояния, ColorSupplier просто используется в классе Lottery